aws/resource_aws_spot_fleet_request.go

```diff
@@ -273,11 +280,10 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 				},
 				Set: hashLaunchSpecification,
 			},
-			// Everything on a spot fleet is ForceNew except target_capacity
 			"target_capacity": {
 				Type:     schema.TypeInt,
 				Required: true,
-				ForceNew: false,
```

Comment:
🙁 I'm not sure why in the original pull request I suggested removing the `Update` function, because it certainly is used for `excess_capacity_termination_policy` and `target_capacity` updates looking at this reworked pull request. I must have just saw the commented out `conn` line with passing tests and assumed the `conn.ModifySpotFleetRequest()` wasn't used? Mea culpa. 🤕 

In any event, it seems like we should keep allowing updates for those two attributes and probably add acceptance testing to prevent introducing any regressions for the existing functionality since the acceptance testing is currently passing when it shouldn't be.

Reply:
Thanks for catching this. I reverted the changes and added test where necessary.